data-api-builder/run-using-container.md

```diff
@@ -33,3 +33,6 @@ docker compose -f "./docker-compose.yml" up
 ```
 
 When using your own Docker compose file, make sure you update your docker-compose file to point to the configuration file you want to use.
+
+**NOTE:**
+If running DAB Application through Docker you only see HTTP endpoints (i.e, HTTPS endpoints are not available). Although the HTTP endpoints can be used to access DAB server, but if you would want to make HTTPS call then you may have to configure your container to use ssl by including the certificate and private key files required for SSL/TLS encryption in your Docker container and expose the HTTPS port. There are several applications and libraries as well that can help redirect HTTPS calls to HTTP within a container such as Nginx, Traefix, etc. 
```

Comment:
```suggestion
When running a Data API builder container in Docker, you'll see that only the HTTP endpoint is mapped. If you want your Docker container to support HTTPS for local development, you'll need to provide your own SSL/TLS certificate and private key files required for SSL/TLS encryption and expose the HTTPS port. There are several applications and libraries as well that can help redirect HTTPS calls to HTTP within a container such as Nginx, Traefix, etc. 
```

>There are several applications and libraries as well that can help redirect HTTPS calls to HTTP within a container such as Nginx, Traefix, etc. 

I think this should be  separate paragraph, noting that this is an option that may be desired for production use. as I am seeing anecdotal suggestions against including certs/key files in the container itself https://stackoverflow.com/questions/67745094/how-do-i-enable-https-for-an-api-hosted-in-docker

Reply:
closing as duplicate https://github.com/MicrosoftDocs/data-api-builder-docs-pr/pull/79